scripts/run.py

```diff
@@ -1409,14 +1413,17 @@ def check_clouds(self):
         """
         for all clouds, check if connection to h2o exists, and that h2o is healthy.
         """
-        time.sleep(3)
-        print("Checking cloud health...")
-        for c in self.clouds:
-            if self._h2o_exists_and_healthy(c.get_ip(), c.get_port()):
-                print("Node {} healthy.".format(c))
-            else:
-                print("Node with IP {} and port {} NOT HEALTHY" .format(c.get_ip(),c.get_port()))
-                # should an exception be thrown?
+        for i in range(5):
```

Comment:
Why do you want to perform the health checks 5 times instead of just once?  Will make the tests run longer.